### PR TITLE
PYTHONPATH update for macOS

### PR DESCRIPTION
**Pull request by Baaaaam**
update documentation for PYTHONPATH on macos

### CODE REVIEW

**Comment by Baaaaam** [comment]
this only make sense if cylus/cymetric#114 is merged

File: INSTALL.rst [hunk 1]
```diff
@@ -56,6 +56,15 @@ You need to add ``~/.local/bin`` to the front of your ``$PATH``:
   echo 'export PATH="$HOME/.local/bin:$PATH' >> ~/.bashrc
   source ~/.bashrc
 
+On MacOSX you also need to add ~/.local/lib/pythonX.Y/site-packages to your
```
Comment (gonuke) [inline INSTALL.rst]: Is this really only for MacOSX?  Or is just a good idea for all OS's anyway.

File: README.rst [hunk 2]
```diff
@@ -83,8 +83,10 @@ on the source code from github is fairly straightforward:
 - to install Cyclus locally (in ``~/.local/``) just run: ``python install.py``
   from the ``cyclus`` folder,
 
-- finally, add the following Cyclus installation path (``~/.local/cyclus``) to
+- finally, add the following Cyclus installation path (``~/.local/bin``) to
   the **front** on your ``$PATH``.
+- additionnaly if you are using MacOSX, also add
+  ``~/.local/lib/pythonX.Y/site-packages`` to your ``$PYTHONPATH``
```
Comment (gonuke) [inline README.rst]: ibid

File: INSTALL.rst (hunk 1 above)
Comment (Baaaaam) [inline INSTALL.rst]: I tested it on ubuntu 16.04 +python3 don't need to define any additional environment variable

**Comment by Baaaaam** [comment]
@gonuke this need to be reviewed as well.

apparently on ubuntu 16.04 (with python3) I do not need to add `~/.local/lib/.../site-packages` to the PYTHONPATH to import Cymetric python lib.

**Comment by gonuke** [comment]
Thanks for this @Baaaaam

**Pull request merged**